keps/sig-scheduling/1258-default-pod-topology-spread/README.md

```diff
@@ -259,13 +272,200 @@ To ensure this feature to be rolled out in high quality. Following tests are man
 - [x] Score extension point implementation. Add support for `maxSkew`.
 - [x] Filter extension point implementation.
 - [x] Disabling `SelectorSpread` when the feature is enabled.
-- [x] Unit, Integration and benchmark test cases mentioned in the [Test Plan](#test-plan).
+- [x] Unit and benchmark test cases mentioned in the [Test Plan](#test-plan).
+
+#### Beta (v1.20):
+
+- [ ] Finalize implementation:
+  - [ ] Map `SelectorSpreadingPriority` to `PodTopologySpread` when using Policy API.
+  - [ ] Provide knob for disabling the k8s default constraints.
+- [ ] Integration tests.
+- [ ] Verify conformance tests passing.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: `DefaultPodTopologySpread`
+    - Components depending on the feature gate: `kube-scheduler`
+  - [x] Other
+    - Describe the mechanism:
+
+      Explicitly disable default spreading constraints for the `PodTopologySpread` plugin in the kube-scheduler config (passed via `--config` command line flag):
+
+      ```yaml
+      apiVersion: kubescheduler.config.k8s.io/v1beta1
+      kind: KubeSchedulerConfiguration
+      profiles:
+        - pluginConfig:
+          - name: PodTopologySpread
+            args:
+              disableDefaultConstraints: true
+      ```
+
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+
+      Only kube-scheduler needs to be restarted.
+
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+      No
+
+* **Does enabling the feature change any default behavior?**
+
+  Yes. Users might experience more spreading of Pods among Nodes and Zones in certain topology distributions.
+  In particular, this will be more noticeable in clusters with more than 100 nodes.
+
+  The specific configuration was chosen to produce a behavior that closely resembles the `SelectorSpread` plugin.
+  See [this PR description](https://github.com/kubernetes/kubernetes/pull/91793) for simulation data.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+
+  Yes. Once disabled, only scheduling of new Pods will be affected.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+
+  Only scheduling of new Pods is affected.
+
+* **Are there any tests for feature enablement/disablement?**
+
+  There are unit tests in `pkg/scheduler/algorithmprovider/registry_test.go` that exercise the configuration of `kube-scheduler` with the plugins that correspond to the Feature Gate enablement.
```

Comment:
They are useful, but these aren't purely enablement/disablement - they are checking the configuration.

Reply:
The feature enablement/disablement leads to a different configuration.

Let me know if the new wording is more clear, or you are asking for other tests.